src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/util/StorageStubHttpClient.java

```diff
@@ -1,19 +1,24 @@
 package uk.gov.hmcts.reform.blobrouter.util;
 
 import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeaders;
 import com.azure.core.http.HttpRequest;
 import com.azure.core.http.HttpResponse;
 import com.azure.core.test.http.MockHttpResponse;
 import org.junit.jupiter.api.Assertions;
 import reactor.core.publisher.Mono;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import static uk.gov.hmcts.reform.blobrouter.util.ResourceFilesHelper.getFileContents;
 
 class StorageStubHttpClient implements HttpClient {
 
     private static final String LIST_CONTAINERS = "?comp=list";
     private static final String LIST_ONE_BLOB = "/bulkscan?include=&restype=container&comp=list";
     private static final String LIST_ZERO_BLOBS = "/empty?include=&restype=container&comp=list";
+    private static final String UPLOAD_NEW_BLOB = "/bulkscan/new.blob?null";
```

Comment:
null ? Could you please explain ?

Reply:
not sure. when running without null test informs me this particular url is not configured. strange serialisation 🤷‍♂